tests/unittests/BackendTestUtils2.cpp

```diff
@@ -978,22 +1054,22 @@ void inferExtract3D(Tensor *input, Tensor *out, llvm::StringRef kind) {
   auto *result = F->createSave("ret", e);
   auto *resultTensor = bindings.allocate(result->getPlaceholder());
 
-  EE.compile(CompilationMode::Infer, F);
+  EE.compile(CompilationMode::Infer);
 
-  updateInputPlaceholders(bindings, {inputs}, {input});
+  updateInputPlaceholders2(bindings, {inputs}, {input});
   EE.run(bindings);
   out->assign(resultTensor);
 }
 
 void inferMaxSplat(Tensor *input, Tensor *out, llvm::StringRef kind) {
   PlaceholderBindings bindings;
-  ExecutionEngine EE(kind);
+  ExecutionEngine2 EE(kind);
   auto &mod = EE.getModule();
   Function *F = mod.createFunction("main");
 
-  auto T = mod.uniqueType(ElemKind::Int8QTy, input->getType().dims(),
-                          2 * input->getType().getScale(),
-                          -input->getType().getOffset());
+  auto T =
+      mod.uniqueType(ElemKind::Int8QTy, input->getType().dims(),
+                     input->getType().getScale(), input->getType().getOffset());
```

Comment:
Why did you change these?

Reply:
It was related to BackendCorrectness, I'll back it out for this PR and revisit in the BackendCorrectnessTest PR.